base_external_dbsource_importer/views/base_external_dbsource_view.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_dbsource_form" model="ir.ui.view">
+        <field name="name">External Dbsource Importer View</field>
+        <field name="model">base.external.dbsource</field>
+        <field name="inherit_id" ref="base_external_dbsource.view_dbsource_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='connector']/../../.." position="inside">
```

Comment:
If you want, you can create a IMP pull request to define name in the groups of `base_external_dbsource` to make a better selector.